spec/models/case/sar/offender_spec.rb

```diff
@@ -290,6 +290,13 @@
     end
   end
 
+  describe "when creating a rejected case" do
```

Comment:
Thought about adding a test to check if it returns nil for a non rejected case, but not sure whether thats necessary.